### PR TITLE
use vanilla appstudio bundle for PaC builds

### PR DESCRIPTION
**Pull request by gabemontero**
let's see how this works @stuartwdouglas @Michkov

### CODE REVIEW

File: .tekton/controller-push.yaml [hunk 1]
```diff
@@ -20,15 +20,15 @@ spec:
...
+        sed -i
         -e 's|\(https://github.com/redhat-appstudio/jvm-build-service/.*?ref=\)\(.*\)|\1{{ revision }}|'
         -e 's/\(newTag: \).*/\1{{ revision }}/'
         components/hacbs/jvm-build-service/base/kustomization.yaml
 
         ./hack/generate-apiexport-overlays.sh components/hacbs/jvm-build-service/
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:devel
```
Comment (Michkov) [inline .tekton/controller-push.yaml:31]: do not change controller, it has infra-deployment update - and it's not java project

**Comment by gabemontero** [comment]
so @Michkov from the build-request-processor build that just failed https://console-openshift-console.apps.appstudio-stage.x99m.p1.openshiftapps.com/k8s/ns/jvm-build-service/tekton.dev~v1beta1~PipelineRun/jvm-build-request-processor-on-pull-request-gm99h/logs/build-container

```
step-inject-sbom-and-push
b6603ebdbeb9254de536dc99eaa9f6b6cc77ea840e92ae5f6e48765d4f1731fb
Getting image source signatures
Copying blob sha256:dff9f8de74c048baca326e1d79db28e7615e8e96497c630177c53e61bbd2f906
Copying blob sha256:e34e3bdec2768c1b351fdcc47094e9e345a7e1c2115d667549e76a02ed282d3c
Copying blob sha256:7cda6ce113beb1259d6bfcc07aec67b8bfe7bde8dba632c478befb5ff9c3f49a
Copying blob sha256:a420a06d40c84759d01e836ed0339af6fbc8cacf4b607fafc62d8d3ad3e533e4
Copying blob sha256:642096fe80428a5438635410d8ee5f8fc1d08258ea5a1d8663c018dae844a190
Copying config sha256:64bf20ea74c770986cbfe85fb3d09e45db42d541dea5c2186c25c1d8ee211b75
Writing manifest to image destination
Storing signatures
64bf20ea74c770986cbfe85fb3d09e45db42d541dea5c2186c25c1d8ee211b75
Getting image source signatures
Copying blob sha256:a420a06d40c84759d01e836ed0339af6fbc8cacf4b607fafc62d8d3ad3e533e4
Copying blob sha256:7cda6ce113beb1259d6bfcc07aec67b8bfe7bde8dba632c478befb5ff9c3f49a
Copying blob sha256:dff9f8de74c048baca326e1d79db28e7615e8e96497c630177c53e61bbd2f906
Copying blob sha256:e34e3bdec2768c1b351fdcc47094e9e345a7e1c2115d667549e76a02ed282d3c
Copying blob sha256:642096fe80428a5438635410d8ee5f8fc1d08258ea5a1d8663c018dae844a190
writing blob: initiating layer upload to /v2/redhat-appstudio/pull-request-builds/blobs/uploads/ in quay.io: unauthorized: access to the requested resource is not authorized
```

can we make changes to build-definitions so that push secret is available to the appstudio bundle?

File: .tekton/controller-push.yaml (hunk 1 above)
Comment (gabemontero) [reply .tekton/controller-push.yaml]: controller no longer changed

**Comment by Michkov** [comment]
@gabemontero add workspace with secret like in https://github.com/redhat-appstudio/jvm-build-service/pull/304/files

File: .tekton/build-request-processor-push.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ spec:
       value: build-request-processor/src/main/docker/Dockerfile.all-in-one
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:devel
   workspaces:
     - name: workspace
```
Comment (Michkov) [inline .tekton/build-request-processor-push.yaml:25]: registry-auth should be here

File: .tekton/cache-push.yaml [hunk 3]
```diff
@@ -20,7 +20,7 @@ spec:
       value: cache/src/main/docker/Dockerfile.all-in-one
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio/build-templates-bundle:devel
   workspaces:
     - name: workspace
```
Comment (Michkov) [inline .tekton/cache-push.yaml:25]: registry-auth should be here

**Review by Michkov — APPROVED** [review]

**Pull request merged**